AUTHORS

```diff
@@ -910,19 +913,21 @@ Jeff Harris <techjeffharris@gmail.com>
 Alexander Makarenko <estliberitas@gmail.com>
 Drew Folta <drew@folta.net>
 Myles Borins <myles.borins@gmail.com>
+Jimb Esser <wasteland@gmail.com>
 Dmitriy Lazarev <wKich@users.noreply.github.com>
 Adam Langley <agl@google.com>
 Kári Tristan Helgason <kthelgason@gmail.com>
+ChALkeR <chalkerx@gmail.com>
 Manuel Valls <manolo@vlrz.es>
 Prayag Verma <prayag.verma@gmail.com>
 Gibson Fahnestock <gib@uk.ibm.com>
 Alan Cohen <alan@ripple.com>
-Christophe Naud-Dulude <christophe.naud.dulude@gmail.com>
-Matthias Bastian <dev@matthias-bastian.de>
+Chris911 <christophe.naud.dulude@gmail.com>
```

Comment:
Is this change on a purpose? Ref: https://github.com/nodejs/node/pull/6373#discussion_r61096600